spec/integration/models/project_spec.rb

```diff
@@ -163,7 +163,7 @@
   describe '#export_clone' do
     context 'when exclude_schedule is true' do
       let(:options) { { exclude_schedules: true } }
-      let(:clone) { GoodData::Project.create(title: 'project clone', client: @client, auth_token: ConnectionHelper::GD_PROJECT_TOKEN) }
+      let(:clone) { GoodData::Project.create(title: 'project clone', client: @client, auth_token: ConnectionHelper::SECRETS[:gd_project_token]) }
```

Comment:
warning: Similar code found in project_spec.rb:166, project_spec.rb:207 (mass = 32) (flay)